inc/cleantalk-settings.php

```diff
@@ -1564,6 +1567,10 @@ function apbct_settings__field__action_buttons()
             echo $link . '&nbsp;&nbsp;&nbsp;&nbsp;';
         }
         echo '</div>';
+    } elseif ( apbct__is_hosting_license() ) {
```

Comment:
We hide the support link for any non-hosting licensed users, am I right?

Reply:
Nope. The support link for the rest users is on line 1555 of this file.